tests/staticfiles_tests/test_management.py

```diff
@@ -508,28 +508,30 @@ class TestCollectionOverwriteWarning(CollectionTestCase):
     # looking for was emitted.
     warning_string = "Found another file"
 
-    def _collectstatic_output(self, **kwargs):
+    def _collectstatic_output(self, verbosity=3, **kwargs):
         """
-        Run collectstatic, and capture and return the output. We want to run
-        the command at highest verbosity, which is why we can't
-        just call e.g. BaseCollectionTestCase.run_collectstatic()
+        Run collectstatic, and capture and return the output.
         """
         out = StringIO()
         call_command(
-            "collectstatic", interactive=False, verbosity=3, stdout=out, **kwargs
+            "collectstatic",
+            interactive=False,
+            verbosity=verbosity,
+            stdout=out,
+            **kwargs,
         )
         return out.getvalue()
 
     def test_no_warning(self):
         """
-        There isn't a warning if there isn't a duplicate destination.
+        There isn't a warning if there isn't a duplicate file.
```

Comment:
The files may not be duplicates; it's the _destination paths_ that conflict. So I think we should revert this
```suggestion
        There isn't a warning if there isn't a duplicate destination.
```